src/users.ts

```diff
@@ -1,24 +1,48 @@
-import {RequestHelper} from './types';
+import {JSONRequest} from './types';
 
-export interface UserObject {
+export type Email = {
+    email: string;
+    primary: boolean;
+};
+
+export interface User {
     id: string;
-    first_name: string;
-    last_name: string;
-    image_id: string;
-    email: [
-        {
-            email: string;
-            primary: true;
-        }
-    ];
+    firstName: string;
+    lastName: string;
+    imageId: string;
+    email: Email[];
 }
 
-export interface UserCollection {
-    [key: string]: () => Promise<UserObject>;
+export interface UserMethods {
+    me: () => Promise<User>;
 }
 
-export default (request: RequestHelper): UserCollection => {
-    const me = () => request('/users/me/') as Promise<UserObject>;
+const SNAKE_CASE_MATCH = /_\w/g;
+const snakeToCamel = (str: string) =>
+    str.replace(SNAKE_CASE_MATCH, (chars: string) => chars[1].toUpperCase());
+
+const transformKeysSnakeToCamel = (obj: {}): {} =>
```

Comment:
I will look into a package, however that one will not work because it also converts dotCase to camel, which breaks our API. That is what led us to making this function internally.

Reply:
This is turning out to be difficult to find something for since it is very generic, but all of them convert `expand.destination_event` to `expandDestinationEvent`, which is undesired.

Reply:
What do you think about leaving this and adding tests? It is that or I make a package of it 😄

Reply:
I moved this around and added tests. Also working on adding something to camel-keys to allow us to be able to remove this.